public/app/features/browse-dashboards/BrowseDashboardsPage.tsx

```diff
@@ -47,7 +47,19 @@ const BrowseDashboardsPage = memo(({ match }: Props) => {
   }, [isSearching, searchState.result, stateManager]);
 
   const { data: folderDTO } = useGetFolderQuery(folderUID ?? skipToken);
-  const navModel = useMemo(() => (folderDTO ? buildNavModel(folderDTO) : undefined), [folderDTO]);
+  const navModel = useMemo(() => {
+    if (!folderDTO) {
+      return undefined;
+    }
+    const model = buildNavModel(folderDTO);
+
+    // Set the first tab ("Dashboards") as active
+    if (model.children) {
+      model.children[0].active = true;
+    }
```

Comment:
Can we .find() by ID to find the dashboards tab? That'll force us to handle if it's missing for whatever reason also.

Reply:
i haven't actually handled it, because i'm not sure what we'd do in that case 😂 

it's not _really_ possible since the result from `buildNavModel` contains that child always.

Reply:
Just don't mark a tab as active - would much rather that than `cannot read property 'active' of undefined`

Reply:
similar, https://github.com/grafana/grafana/pull/68249